homeassistant/components/growatt_server/sensor.py

```diff
@@ -504,6 +708,45 @@ def update(self):
                     self.plant_id, self.device_id
                 )
                 self.data = {**storage_info_detail, **storage_energy_overview}
+            elif self.growatt_type == "mix":
+                mix_info = self.api.mix_info(self.device_id)
+                mix_totals = self.api.mix_totals(self.device_id, self.plant_id)
+                mix_system_status = self.api.mix_system_status(
+                    self.device_id, self.plant_id
+                )
+
+                mix_detail = self.api.mix_detail(
+                    self.device_id, self.plant_id, date=datetime.datetime.now()
+                )
+                # Get the chart data and work out the time of the last entry, use this as the last time data was published to the Growatt Server
+                mix_chart_entries = mix_detail["chartData"]
+                sorted_keys = sorted(mix_chart_entries)
+
+                # Create datetime from the latest entry
+                date_now = dt.now().date()
+                last_updated_time = dt.parse_time(str(sorted_keys[-1]))
+                combined_timestamp = datetime.datetime.combine(
+                    date_now, last_updated_time
+                )
+                # Convert datetime to UTC
+                combined_timestamp_utc = dt.as_utc(combined_timestamp)
+                mix_detail["lastdataupdate"] = combined_timestamp_utc
```

Comment:
```suggestion
                mix_detail["lastdataupdate"] = combined_timestamp_utc.isoformat()
```

Reply:
You happy if I do this one as an amend commit? Keeps it nice and tidy?
The alternative is that I just accept your suggestion and let github do it's thing.

Reply:
Easiest is to let github UI commit it. Then I don't have to review again either.